e2e/login.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { test, expect } from '@playwright/test';
+
+// Get data from config about projects and convert to JS object setting TS types
+
+const projects = JSON.parse(process.env.PROJECTS!) as Record<
+  string,
+  { url: string; app: string; title: string }
+>;
+
+// Get arrays of urls, apps, titles from that object using just one loop with reduce method
+
+const { urls, apps, titles } = Object.values(projects).reduce(
+  (acc: { urls: string[]; apps: string[]; titles: string[] }, project) => {
+    acc.urls.push(project.url);
+    acc.apps.push(project.app);
+    acc.titles.push(project.title);
+    return acc;
+  },
+  { urls: [], apps: [], titles: [] },
+);
+
+// Create old-school loop to loop over all the projects and create a test for everyone
+
+for (let i = 0; i < urls.length; i++) {
```

Comment:
Please iterate over the projects itself, for example `for (project of projects)`

Reply:
done